aspnetcore/mvc/models/validation.md

```diff
@@ -76,13 +76,19 @@ Client-side validation requires a value for a form field that corresponds to a m
 
 Model state represents validation errors in submitted HTML form values.
 
-MVC will continue validating fields until reaches the maximum number of errors (200 by default). You can configure this number by inserting the following code into the `ConfigureServices` method in the *Startup.cs* file:
+MVC will continue validating fields until it reaches the maximum number of errors (200 by default). You can configure this number with the following code in `Startup.ConfigureServices`:
```

Comment:
MVC has an additional option which will stop validation before it's done: `MvcOptions.MaxValidationDepth` constrains the visitor to recursing through the model only to the specified depth.  The default value is `200` when running with `CompatibilityVersion.Version_2_2` or later and `null` i.e. unconstrained when running with an earlier `CompatibilityVersion`.